lib/project_types/extension/models/specification_handlers/checkout_ui_extension.rb

```diff
@@ -62,6 +62,15 @@ def localization(context)
           end
         end
 
+        def read_locale_file(filename)
+          Base64.strict_encode64(File.read(filename, mode: "rt", encoding: "UTF-8").strip.sub("\xEF\xBB\xBF", ""))
```

Comment:
Did you consider using this magical `.read()` [mode](https://stackoverflow.com/questions/5011504/is-there-a-way-to-remove-the-bom-from-a-utf-8-encoded-file) instead? The example is:

```ruby
File.open('file.txt', "r:bom|utf-8")
```

Reply:
Need to try it.